src/behavior/src/A32NX_Interior_ATC.xml

```diff
@@ -223,14 +223,13 @@
                     <ANIM_TYPE>KNOB</ANIM_TYPE>
                     <ANIM_TEMPLATE>ASOBO_GT_Switch_2States</ANIM_TEMPLATE>
                     <SWITCH_DIRECTION>Horizontal</SWITCH_DIRECTION>
-                    <SWITCH_POSITION_TYPE>L</SWITCH_POSITION_TYPE>
-                    <SWITCH_POSITION_VAR>A32NX_SWITCH_ATC_ALT</SWITCH_POSITION_VAR>
```

Comment:
It's not about renaming the variable. It's that A32NX_SWITCH_ALT_RPTG did not exhibit the same behavior than what A32NX_SWITCH_ATL_ALT did previously, in that changing the var does not make the cockpit switch reflect the correct position.
I wonder if removing these two lines is what actually caused the cockpit switch glitch - but I'm really not familiar with how knob animation ties into LVARS, so it might just be me.

Before this change, setting A32NX_SWITCH_ATC_ALT to 0 via i.e. spad.next caused the cockpit switch to flip into the "off" position.
After your changes, setting A32NX_TRANSPONDER_ALT_RPTG to 0 did not cause the cockpit switch to flip into "off", it would remain in "on". That's all I'm saying - it's no problem if the variable name changes, as long as it still properly interacts with the cockpit knob.
Not sure if it's clear this way.
I haven't tested your latest change yet, so maybe that resolves this. Once this is available on exp, I can give it a shot.

Reply:
I didn't actually push the correct change to the repo, it's been pushed now.

Reply:
Thanks and sorry for the extra work. I'll give it a shot once it is available on experimental or development.